docs/assets/js/main.js

```diff
@@ -55,6 +55,7 @@ function navigateBack() {
 
     if (referrer.includes(currentHost) && window.history.length > 1) {
         window.history.back();
+        toggleHeaderMenu();
```

Comment:
While implementing I found another similar bug that when the back button was clicked on the playbook section the LHN was not closed, so added this here to fix that.

Reply:
I am seeing a flicker on ios safari on back button click. I think this change is causing the below flicker on ios but still not closing the LHN on back button. If the flicker due to this change, I would suggest to remove it(If not able to fix the flicker) since this is an unrelated change to the issue actually.

@Talha345  

https://github.com/Expensify/App/assets/46707890/a8274bba-ffc0-4264-ab1d-fa06e1f3effd

Reply:
@abdulrahuman5196 Yes this change was causing this issue on IOS since the behaviour seems to be a bit different on Safari and Chrome so I have reverted this change since it was unrelated to this issue and caused side-effect.